scripts/ci/run-jenkins.sh

```diff
@@ -252,3 +252,14 @@ elif [[ ${CI_TAGS} == *'mac-sdk'* ]];                  then ${MONO_REPO_ROOT}/sc
 elif [[ ${CI_TAGS} == *'no-tests'* ]];                 then exit 0;
 else make check-ci;
 fi
+
+if [[ ${CI_TAGS} == *'win-'* ]];
+then
+	echo "Skipping telemetry phase"
+else
+	export MONO_SENTRY_URL="https://6c9dc1505a634a2292cbc60c7eda2a46@sentry.io/1267889"
+	export MONO_SENTRY_OS="placeholder-arch"
+	${TESTCMD} --label=sentry-telemetry-upload --timeout=2m make -C mcs/class/corlib upload-all
```

Comment:
We cannot call `upload-all` in all subdirectories we care about, otherwise the exhaustive list will be huge.